understanding/conformance.html

```diff
@@ -156,7 +156,7 @@ <h3>Understanding Requirement 3</h3>
          <h3>Understanding Requirement 4</h3>
          
       	<blockquote class="scquote" id="cc4">
-      		<p><strong>Only Accessibility-Supported Ways of Using Technologies</strong>: Only <a>accessibility-supported</a> ways of using <a>technologies</a> are <a>relied upon</a> to satisfy the success criteria. Any information or functionality that is provided in a way that is not accessibility supported is also available in a way that is accessibility supported. (See <a href="https://www.w3.org/WAI/WCAG21/Understanding/conformance#accessibility-support">Understanding accessibility support</a>.)</p></blockquote>
+      		<p><strong>Only Accessibility-Supported Ways of Using Technologies</strong>: Only <a>accessibility-supported</a> ways of using <a>technologies</a> are <a>relied upon</a> to satisfy the Success Criteria. Any information or functionality that is provided in a way that is not accessibility supported is also available in a way that is accessibility supported. (See <a href="https://www.w3.org/WAI/WCAG21/Understanding/conformance#accessibility-support">Understanding accessibility support</a>.)</p></blockquote>
```

Comment:
Should the link at the end of this sentence be generalized?

Reply:
Done